.gitlab-ci.yml

```diff
@@ -0,0 +1,21 @@
+image: ubuntu:bionic
+
+variables:
+  GRADLE_VERSION: "5.4.1"
+  
+before_script:
+  - apt-get update && apt-get -y upgrade
+  - apt-get -y install openjdk-11-jdk gradle
+  - gradle wrapper --gradle-version=$GRADLE_VERSION
+  - chmod +x ./gradlew
+
+build:
+  script:
+    - java -version
+    - LC_ALL=C.UTF-8 ./gradlew build
+  artifacts:
+    paths:
+      - core/build
```

Comment:
What does Gitlab do with these artifacts?

Reply:
Makes the available for download. (On commercial projects with private gitlab, I've used the CI-built artifacts as the official build binaries)

Reply:
Ok. I'm not sure if we want binary downloads on GitLib (or GitHub) however. Our website downloads currently point at maven.org.

Reply:
Yeah, we can turn them off if we want. I also sometimes configure it so that other output files such as logs or debug/error information is preserved. They're not super-easy to find on the GitLab site, so I'm not too worried that people will get confused by them and use them instead of the official Maven binaries.

Reply:
For a first step, I think I'd like to keep things simple and omit this part.

Reply:
Fixed in force-push.